index.ts

```diff
@@ -21,6 +21,7 @@ export function bootstrap(callback: Function, options = <any> {}) {
   const script = document.createElement('script');
   script.type = 'text/javascript';
   script.src = options.url;
+  script.dataset['esriLoader'] = '';
```

Comment:
I'd like to think of something better than an empty string to use for the value, maybe `true` or